core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+Specific Terms and Conditions (“Specific T&C”) defines the terms and conditions governing a specific product, service or asset or set thereof offered and delivered by Provider(s) to Requester(s).
+
+**[D2]**	Each commercial state object SHOULD have its own Specific T&C document.
+
+This would allow the fine graining and consistent application of commercial state object specific business rules and data.
+
+### 2.4.2	Commercial Documents 
+
+Commercial Documents, a category of commercial state objects, refer to a specific product/service/asset or set thereof, which may or may not be modified from an original offering to meet the Requester requirements and includes operational and commercial details. A commercial document is an abstract construct representing mutual commitments.
+
+**[R4]**	A commercial state object to be transacted on MUST be based on a specific commercial document.
+
+**[R5]**	A commercial document MUST be represented as an electronic record. 
+
+**[R6]**	A commercial document MUST be authorized by legal representatives of the parties, or their legal delegates.
+
+**[D3]**	The definition of a commercial document authorization SHOULD be stated in the legal agreement underlying the commercial document.
+
+Authorizations for commercial transactions are a foundational element in the conext of this document, as they are in paper based agreements. Therefore, any legal authorization agreements relevant to the commercial agreement between commercial counterparties, and thus to commercial transactions between them, are important to be mapped onto a BPI to ensure mitigating the risk of unauthorized signatures.
+
+**[R7]**	The representatives and their authorized delegates who can perform commercial document authorizations SHOULD be explicitly listed or inferred from the stated legal delegation rules of the counterparties in the contract underlying any commercial document. 
+
+**[R8]**	A commercial document MUST be non-repudiable.
+
+Note that while non-repudiation in the physical world is most often tied to a physical signature of an individual on a legal document, in the digital world a digital signature over a digital legal document such as an Order or an Invoice belonging to a known and verifiable digital identity of a counter party serves the same purpose.
+
+Example
+A Buyer and Seller may agree that a signed Order requires a signed original paper copy, or a digitally signed electronic Order Form, in addition to an Order being a digitally signed and recorded within a Baseline Protocol Implementation.
+
+## 2.5 Distributed Ledger Technology (DLT)
+
+A distributed ledger (also called a shared ledger or distributed ledger technology or DLT) is a consensus of replicated, shared, and synchronized digital data spread across multiple sites. There is no central administrator or centralized data storage. 
```

Comment:
Maybe consider capitalising 'Distributed Ledger Technology' so if people are skimming, it is clear that the following acronym is related)

Reply:
Done. And yes ... you can click on "Create a Review" and then mark a comment as a change request